### PR TITLE
Running test on one file added in README.md

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #875  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by divyabaid16** [comment]
@publiclab/mapknitter-reviewers

File: README.md [hunk 1]
```diff
@@ -227,6 +227,14 @@ By running like this you'll see a lot of warnings and deprecation notices - FOR
 
 `rake test:all`
 
+#### To run one test file:
```
Comment (gauravano) [inline README.md]:
```suggestion
#### Running tests of a specific file:
```

File: README.md [hunk 2]
```diff
@@ -227,6 +227,14 @@ By running like this you'll see a lot of warnings and deprecation notices - FOR
...
+
+`rails test test/unit/some_file.rb`
+
+#### And to run just a single test within a file:
```
Comment (gauravano) [inline README.md]:
```suggestion
#### Running a single test from the test suite:
```

**Comment by divyabaid16** [comment]
I have made the changes @gauravano.
Thanks!

**Pull request merged**